wallets/react/src/provider.tsx

```diff
@@ -74,9 +79,30 @@ function Provider(props: ProviderProps) {
         }
       });
 
-      if (props.autoConnect) clearPersistance();
+      if (props.autoConnect) {
+        clearPersistance();
+      }
       return await Promise.allSettled(disconnect_promises);
     },
+
+    async suggestAndConnect(type, network) {
+      const wallet = wallets.get(type);
+      if (!wallet) {
+        throw new Error(`You should add ${type} to provider first.`);
+      }
+      const walletInstance = getWalletInstance(wallet);
+      const result = await walletInstance.suggestAndConnect(network);
+      if (props.autoConnect) {
+        tryPersistWallet({
+          type,
+          walletActions: wallet.actions,
+          getState: api.state,
+        });
+      }
```

Comment:
No need to persist the wallet because it is already connected

Reply:
Done